RELEASES.md

```diff
@@ -0,0 +1,19 @@
+# Creating a new release (guidelines for repo-admins)
+
+After review of a protocol-branch and before each release the following steps are necessary:
+
+
+1. with the protocol-branch checked-out:
+    1. bump the version number in the `index.Rmd` yaml section from `yyyy.nn.dev` to `YYYY.NN`. Some examples:
+        - a new protocol added in 2020 that was released in 2020 (fifth release in that year): `2020.00.dev` to `2020.05`
+        - an update of the `2020.05` protocol in 2021 (first release in that year): `2020.05.dev` to `2021.01`
+    1. update the `.zenodo.json` file:
+        - add new authorships
+    1. update the repo `NEWS.md` file
+1. merge the branch to the master and add general and specific tags (see [release model](README.md#release-model)):
+    1. general tag: `protocols-YYYY.NN`
+    1. specific tag: `protocol-code-YYYY.NN`
```

Comment:
An idea for the protocolhelper package, because this step, and step 1 (version number update) is prone to errors? I.e. one or more functions to:

- determine and print the next protocols tag
- determine and print the next protocol-specific tag(s), i.e. based on the occurrence of dev versions in the working directory of current checked out branch
- (optionally:) bump version number of the involved protocol(s) accordingly, and return message(s) on this

Reply:
Good ideas. I've made an issue of your comment inbo/protocolhelper#27

Reply:
Done in 65687c6